mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(amount) {
+  return (amount * 20) / 100 + amount;
```

Comment:
this doesn't seem quite right. This function should just calculate the sales tax (not add the sales tax). 

```
return amount * 0.2
```